Library/ViewModels/PostCampaignCheckoutViewModelTests.swift

```diff
@@ -455,7 +455,7 @@ final class PostCampaignCheckoutViewModelTests: TestCase {
       self.scheduler.run()
 
       self.goToApplePayPaymentAuthorization.assertDidNotEmitValue()
-      self.processingViewIsHidden.assertValues([false, true])
+      self.processingViewIsHidden.assertValues([true, false])
```

Comment:
Was this intentional? Surprised to see this swapped

Reply:
whoops. I can fix this. Although the tests all passed, so that's interesting...